Source/DOM classes/SVG-DOM/SVGHelperUtilities.m

```diff
@@ -502,8 +502,6 @@ +(CALayer *) newCALayerForPathBasedSVGElement:(SVGElement<SVGTransformable>*) sv
 		maskLayer.fillColor = [UIColor blackColor].CGColor;
 		maskLayer.strokeColor = nil;
 		gradientLayer.mask = maskLayer;
-		if ( [gradientLayer.type isEqualToString:kExt_CAGradientLayerRadial])
```

Comment:
See changes in `SVGLinearGradientElement`, which use [kCAGradientLayerAxial](https://developer.apple.com/documentation/quartzcore/kcagradientlayeraxial?language=objc). And changes in `SVGGradientElement`, which use [kCAGradientLayerRadial](https://developer.apple.com/documentation/quartzcore/kcagradientlayerradial?language=objc)

This value previouslly is a user-defined value, but it break the support for `SVGLayeredImageView` rendering `<radialGradient>` element. Because QuarzCore does not recognize it as a valid type in [CAGradientLayerType](https://developer.apple.com/documentation/quartzcore/cagradientlayertype?language=objc)